### PR TITLE
Fix/search

### PR DESCRIPTION
**Pull request by leticiawanderley**
#### What is the purpose of this pull request?
Fix search transition functionality on desktop view.

#### Types of changes
* [x] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/HomePage.js [hunk 1]
```diff
@@ -10,32 +11,63 @@ class HomePage extends Component {
...
+  }
+
+  watchScrollUpDesktop = () => {
+    const currentScroll = window.scrollY
+    this.setState({ scroll: currentScroll })
+  }
+
   render() {
+    const { scroll } = this.state
+    const jumbontronSize =
```
Comment (nandoacoelho) [inline react/HomePage.js]: Seria bom jogar isso pro state. Fica meio jogado, aqui no render.

File: react/components/Header.js [hunk 2]
```diff
@@ -63,6 +63,11 @@ class Header extends Component {
       window.document &&
       window.document.getElementById('extension-store-header') &&
       window.document.getElementById('extension-store-header').offsetHeight
+    const jumbontronSize =
```
Comment (nandoacoelho) [inline react/components/Header.js]: mesma coisa aqui

File: react/HomePage.js [hunk 3]
```diff
@@ -10,32 +11,63 @@ class HomePage extends Component {
...
-      <div className="w-100 h-100">
-        <div className="flex-ns items-center-ns vh-100-s h-50-ns w-100 bg-serious-black pt9 pl5-s pl7-ns white">
-          <div>
-            <div className="b f2-s f1-ns mt9-s mt0-ns">Extension Store</div>
-            <div className="mt5 mb9-ns f4-s f3-ns fw3">
-              {this.translate('homeText')}
+      <div className="w-100">
+        <div id="jumbotron-home" className="w-100">
+          <div className="flex-ns items-center-ns h-100-s h-50-ns w-100 bg-serious-black pt9 pl5-s pl7-ns white">
+            <div>
```
Comment (kevinch) [inline react/HomePage.js]: Useful `div`?

File: react/components/AppCategory.js [hunk 4]
```diff
@@ -12,13 +12,13 @@ class AppCategory extends Component {
 
   translate = id => this.props.intl.formatMessage({ id: `extensions.${id}` })
 
-  removeSlashes = category => category.replace(/\//g, '')
+  removeSlashes = category => category && category.replace(/\//g, '')
```
Comment (kevinch) [inline react/components/AppCategory.js]: Do you have a `utils.js` (or similar) file? If yes, this should be there.

File: react/components/ConfirmButton.js [hunk 5]
```diff
@@ -12,7 +12,7 @@ class ConfirmButton extends Component {
   render() {
     const { value } = this.props
     return (
-      <div className="bg-rebel-pink tc br2 w-100">
+      <div className="bg-rebel-pink hover-bg-heavy-rebel-pink tc br2 w-100">
         <Button onClick={this.handleClick} block>
           <span className="white">{value}</span>
```
Comment (kevinch) [inline react/components/ConfirmButton.js]: Is this `white` color not against the Styleguide idea? Maybe better to use a `Button` with `primary` or `secondary` attribute.

File: react/HomePage.js (hunk 3 above)
Comment (leticiawanderley) [reply react/HomePage.js]: Sorry, it was missing a `width`

File: react/components/ConfirmButton.js (hunk 5 above)
Comment (leticiawanderley) [reply react/components/ConfirmButton.js]: The store theme has rebel-pink, white and grey buttons, as this is not an admin it is ok to have a colour scheme that doesn't match the styleguide.

**Pull request merged**